blog/2024-05-21-blog-status-of-lbse-in-webkit.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Update on the Layer Based SVG Engine (LBSE) in WebKit"
+author: rbuis
+permalink: /blog/status-of-lbse-in-webkit.html
+thumbnail: /assets/lbse-logo-wide.png
+preview: A look back at the last 7 months of development on the Layer Based SVG Engine (LBSE) in WebKit.
+---
+
+This blog entry gives an update on upstreaming and development of LBSE in WebKit in the last 7 months. For an explanation of
+what LBSE is and how it is related to WPE see this previous <a href="https://wpewebkit.org/blog/05-new-svg-engine.html">entry</a> as a refresher.
+
+<figure>
+  <a href="/assets/svg/wix-logo.svg" target="_blank"><img
+    style="display: block; float: right; margin-left: 20px; mrgin-right: auto; height: 60px;"
+    src="/assets/svg/wix-logo.svg"
+    alt="The Wix logo">
+  </a>
+</figure>
+
+Thanks to the generous funding by <a href="https://www.wix.com" alt="Wix homepage"><b>Wix</b></a>, which extensively uses SVG in their products and has a broad
+knowledge of the SVG peculiarities across browser engines, the past 7 months were crucial for LBSE development. During this period, several advanced SVG painting
```

Comment:
"...knowledge of the SVG peculiarities across browser engines, LBSE made great progress in the past 7 months. During this period, several advanced SVG painting"

Reply:
Done.